java/connectors/semantickernel-connectors-memory-jdbc/src/main/java/com/microsoft/semantickernel/connectors/memory/jdbc/JDBCConnector.java

```diff
@@ -18,43 +18,22 @@
 import reactor.core.publisher.Mono;
 import reactor.core.scheduler.Schedulers;
 
-public class Database {
-
-    static class DatabaseEntry extends DataEntryBase {
-        private final String metadata;
-        private final String embedding;
-
-        public DatabaseEntry(
-                String key, String metadata, String embedding, ZonedDateTime timestamp) {
-            super(key, timestamp);
-            this.metadata = metadata;
-            this.embedding = embedding;
-        }
-
-        public String getMetadata() {
-            return metadata;
-        }
-
-        public String getEmbedding() {
-            return embedding;
-        }
-    }
-
-    // Convenience method to format a ZonedDateTime in a format acceptable to SQLite
-    private static String formatDatetime(ZonedDateTime datetime) {
+public class JDBCConnector implements Connector {
```

Comment:
Is this a JDBC connector or a connector that is specific to Semantic Kernel? If this is the case, would the SemanticKernelConnector be more descriptive?